app/controllers/TagManagementApi.scala

```diff
@@ -51,12 +51,13 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
   }
 
   def searchTags = APIAuthAction { req =>
-    
+
     val criteria = TagSearchCriteria(
       q = req.getQueryString("q"),
       searchField = req.getQueryString("searchField"),
       types = req.getQueryString("types").map(_.split(",").toList),
-      referenceType = req.getQueryString("referenceType")
+      referenceType = req.getQueryString("referenceType"),
+      hasFields = req.getQueryString("hasFields").map(_.split(",").toList)
```

Comment:
Potentially needs a `trim`?

Reply:
Sure, I'll add that to `hasFields` and `types`!